Prospress/action-scheduler-admin
# Disable processed rows after processing

Closes #39 

This PR disables row checkboxes when the row has been either run or cancelled. 

This PR includes the commits from #37. We will hold off on merging this until after #37 is merged.

### Testing instructions

- Use this snippet to create test pending actions
```
add_action( 'init', function() {
	$prefix = [ 'one', 'two', 'three', 'four', 'five' ];
    for ( $i = 1; $i <= 20; $i++ ) {
		$hook = $prefix[ $i % 5 ] . '-single-test-' . $i;
        as_schedule_single_action( time() + DAY_IN_SECONDS, $hook );
	}
} );
```
- Run `npm run install`
- Run `npm run build`
- Go to WooCommerce -> Scheduled Actions
- Switch to 25 rows per page
- Verify selecting & unselecting actions works both individually and with the all box
- Select a pending action
- Run that action
- Repeat with Cancel
- Test that neither action can be individually checked
- Test that the select all checkbox does not check either action
- With all remaining actions selected, click either `Run` or `Cancel`
- Once the actions have been processed verify that the select all checkbox does not select any actions
